include/mamba/core/fetch.hpp

```diff
@@ -66,6 +66,21 @@ namespace mamba
             return m_ignore_failure;
         }
 
+        inline void set_active_logs(bool yes)
```

Comment:
do we need this? Context::verbosity is the same, no?

Reply:
it's needed to silent specifically all the logs emitted in a `DownloadTarget`, since the target can be downloaded later and the possible callback can also emit some logs.

Reply:
I still don't understand ... the logs that I see all use LOG_INFO. what's the issue with that? This looks like introducing another log level for one particular use-case? Is that really required? 

I am happy if you change the logs in the callbacks to be less verbose if that's the goal.

Reply:
Some logs are emitted in `perform` (previously in constructor, moved in this PR) and `set_result`.
There is no point of logging failed downloads when in most cases they are allowed to fail, it's totally misleading.

We could remove this later when having capability to check existence of the target on the server using header request instead.